### PR TITLE
Create list contacts UI

### LINKED ISSUE #6: Create list contacts UI

### PR DESCRIPTION
**Pull request by gaagul**
fixes #6 
Added delete contact alert.

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/utils.jsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    width: "20",
+  },
+  {
+    title: "",
+    dataIndex: "menu",
+    key: "menu",
+    width: "10",
+    render: () => (
+      <div>
+        <Dropdown buttonStyle="text" icon={MenuHorizontal}>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/utils.jsx:49]: What's the point of this `div`?

File: app/javascript/src/components/Dashboard/Notes/utils.js [hunk 2]
```diff
@@ -3,3 +3,9 @@ import dayjs from "lib/dayjs";
 export const calculateCreatedAgo = date => dayjs(date).fromNow();
 export const formatToDayTime = dateTime =>
   dayjs(dateTime).format("dddd, hh:mmA");
+
+export const buildSelectOptions = options =>
+  options.map(option => ({
+    label: option,
+    value: option,
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/utils.js:10]: `value` should be the camelCase version of the `label`.

**Review by yedhink — COMMENTED** [review]
@gaagul

File: app/javascript/src/components/Dashboard/Contacts/utils.jsx (hunk 1 above)
Comment (gaagul) [reply app/javascript/src/components/Dashboard/Contacts/utils.jsx:49]: I will make the necessary changes.

**Pull request merged**